pkg/master/master.go

```diff
@@ -781,11 +782,14 @@ func (m *Master) api_v1() *apiserver.APIGroupVersion {
 func (m *Master) expapi(c *Config) *apiserver.APIGroupVersion {
 	controllerStorage := expcontrolleretcd.NewStorage(c.DatabaseStorage)
 	autoscalerStorage := horizontalpodautoscaleretcd.NewREST(c.DatabaseStorage)
+	nodeMetricsStorage := nodemetrics.NewREST(c.DatabaseStorage)
 
 	storage := map[string]rest.Storage{
 		strings.ToLower("replicationControllers"):       controllerStorage.ReplicationController,
 		strings.ToLower("replicationControllers/scale"): controllerStorage.Scale,
 		strings.ToLower("horizontalpodautoscalers"):     autoscalerStorage,
+		// /api/experimental/nodeMetrics/{myNode}
```

Comment:
The comment was just to remind myself, to be removed once the PR is ready.

`horizontalpodautoscalers` was merged not long ago, see that PR for comments on the naming. Specifically:

https://github.com/kubernetes/kubernetes/pull/12344#discussion_r36701059

I just followed that pattern.